README.md

```diff
@@ -14,9 +14,12 @@
 
 ## Overview
 
-The Asset Tracker Template is a modular framework for developing IoT applications on nRF91-based devices. Built on [nRF Connect SDK](https://www.nordicsemi.com/Products/Development-software/nRF-Connect-SDK) and [Zephyr RTOS](https://docs.zephyrproject.org/latest/), it provides an event-driven architecture for battery-powered IoT use cases with cloud connectivity, location tracking, and sensor data collection.
+The Asset Tracker Template is a modular framework for developing IoT applications on nRF91-based devices.
+It is built on the [nRF Connect SDK](https://www.nordicsemi.com/Products/Development-software/nRF-Connect-SDK) and [Zephyr RTOS](https://docs.zephyrproject.org/latest/), and provides a modular, event-driven architecture suitable for battery-powered IoT use cases.
+The framework supports features such as cloud connectivity, location tracking, and sensor data collection.
 
-The system uses modules that communicate through [zbus](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/services/zbus/index.html) channels for loose coupling and maintainability. It's suitable for asset tracking, environmental monitoring, and other IoT applications requiring modularity and power efficiency.
+The system is organized into modules, each responsible for a specific functionality, such as managing network connectivity, handling cloud communication, or collecting environmental data.
+Modules communicate through [zbus](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/services/zbus/index.html) channels, ensuring loose coupling and maintainability.
```

Comment:
Lets just refer to zephyrs offical documentation here:
https://docs.zephyrproject.org/latest/services/zbus/index.html

Reply:
WADR. I don't understand why we are now routing docs from other sw projects via nrf cloud docs, this doesn't make any sense. The documentation doesn't render properly either in the nordic docs 😢

Reply:
The rendered docs in docs.nordicsemi.com dosen't look very healthy. And I don't quite see why we are routing docs for other sw projects via techdocs? Sounds like double work maintaining docs for other prj as well